scripts/travis.sh

```diff
@@ -38,16 +38,27 @@ if [[ "$action" == "install" ]]; then
   sudo mv $(find /tmp -name "glide") /usr/bin
 elif [[ "$action" == "lint" ]]; then
   echo "================================="
-  echo "             Lint                "
+  echo "              Lint               "
   echo "================================="
-  CMD_PASS=$(gofmt -d $PROJECT_ROOT/cmd 2>&1 | read; echo $?)
-  PKG_PASS=$(gofmt -d $PROJECT_ROOT/pkg 2>&1 | read; echo $?)
-  echo "CMD_PASS=$CMD_PASS"
-  echo "PKG_PASS=$PKG_PASS"
-  FULL_PASS=$([[ $CMD_PASS == 1 ]] && [[ $PKG_PASS == 1 ]]; echo $?)
-  echo "FULL_PASS=$FULL_PASS"
+  # to have lint fail build add -set_exit_status option
+  # to each golint command
+  golint $PROJECT_ROOT/cmd/...
+  golint $PROJECT_ROOT/pkg/...
+  exit $?
+elif [[ "$action" == "format" ]]; then
+  echo "================================="
+  echo "             Format              "
+  echo "================================="
+  if [ -n "$(gofmt -l cmd pkg)" ]; then
```

Comment:
This returns values example
```stat cmd: no such file or directory```
```stat pkg: no such file or directory```

any reason not to always just use $PROJECT_ROOT ?